keyboards/durgod/k320/keymaps/default/keymap.c

```diff
@@ -0,0 +1,418 @@
+/* Copyright 2020 kuenhlee and Don Kjer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Layer shorthand
+enum _layer {
+    _WBL,
+    _WFL,
+    _WSL,
+    _MBL,
+    _MFL,
+    _MSL
+};
+
+#ifndef DEFAULT_LAYOUT
+# define DEFAULT_LAYOUT _WBL // Change to _MBL to boot into Mac layout.
+#endif
+#ifndef ALT_LAYOUT
+# define ALT_LAYOUT     _MBL // Layout to display 'MR' led when active.
+#endif
+
+static bool win_key_locked = false;
+static bool mac_media_locked = false;
+static bool win_appkey_pressed = false;
+static bool mac_fnkey_pressed = false;
+
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    KC_TGUI = SAFE_RANGE,  // Toggle between GUI Lock or Unlock (Windows)
+    KC_TMED,               // Toggle between Fx keys or Media keys (Mac)
+    KC_WFN,                // Windows function key
+#ifndef APPLE_FN_ENABLE    // If no apple fn support, use custom fn key.
+    KC_MFN,                // Mac function key
+#endif
+    MO_WSL,                // Windows system key
+    MO_MSL,                // Mac system key
+    DF_W2MBL,              // Change layout from Windows to Mac
+    DF_M2WBL               // Change layout from Mac to Windows
+};
+
+#ifdef APPLE_FN_ENABLE     // If apple fn support, use as mac fn key.
+# define KC_MFN KC_APPLE_FN
+#endif
+
+// Passthrough GUI desktop control keys if they aren't available.
+#ifndef KC_MCON
+# define KC_MCON KC_TRNS
+#endif
+#ifndef KC_LPAD
+# define KC_LPAD KC_TRNS
```

Comment:
I could be wrong, but lines 56, 61 and 64 might need the space character removed after the `#` for these preprocessor directives.

Reply:
I don't believe so; that is just a stylistic indentation